src/pages/messaging/agent-chat-api/v3.5/index.mdx

```diff
@@ -1624,6 +1628,8 @@ https://api.livechatinc.com/v3.5/agent/action/tag_thread \
 
 ### Untag Thread
 
+Untags a given **thread**. In this method, letter case needs to be respected for tag names.
```

Comment:
I am not sure if I understood correctly xd in this method letter case needs to be respected as well?

Reply:
yes, both tag and untag methods are case-sensitive